examples/experimental/ProductionMonitoring/ProductionMonitoringConceptualOverview.ipynb

```diff
@@ -6,29 +6,63 @@
    "metadata": {},
    "source": [
     "# W&B Production Monitoring Overview\n",
-    "In this notebooks you will learn to monitor your production models with W&B through an illustrative example. We will train a model to correctly identify handwritten digits, then monitor a locally deployed version of the model.\n",
     "\n",
-    "_Note: To keep the example focused on important code, much of the dataset manipulation, modelling, and other utilities are packaged in local files and imported here_"
+    "This notebook demonstrates how to monitor production models with W&B through an illustrative example. We will train a model to correctly identify handwritten digits, then monitor a locally deployed version of the model. We create a gradio app which runs in the notebook and lets a user draw/\"handwrite\" characters with the mouse and give live feedback by labeling the character as the digit 0-9.\n",
+    "\n",
+    "_Note: To keep the example focused on important code, much of the dataset manipulation, modelling, and other utilities are packaged in local files and imported here_\n",
+    "\n",
+    "## Step 0: Setup & import dependencies"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "ad4c25c2",
+   "id": "3af52605",
    "metadata": {},
    "outputs": [],
    "source": [
-    "%load_ext autoreload"
+    "!pip install tensorflow\n",
+    "!pip install gradio\n",
+    "!pip install weave"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "55afcd6c",
+   "metadata": {},
+   "source": [
+    "Log in to W&B to sync these examples to your W&B account, where you can view, interact with, and customize the resulting Tables and Boards."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c9e3e332",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import wandb\n",
+    "wandb.login()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "48354a22",
+   "metadata": {},
+   "source": [
+    "Set your W&B entity (username or team name) and optionally rename the destination project."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "2427bb8e",
+   "id": "424dc619",
    "metadata": {},
    "outputs": [],
    "source": [
-    "%autoreload 2"
+    "WB_ENTITY = \"stacey\"\n",
```

Comment:
I think we remove your entity here.

Reply:
other than this, I think it's ready to merge

Reply:
oops thank you was iterating many times and forgot!